code/mathematical-algorithms/basic/log_of_factorial/logfactorial.c

```diff
@@ -1,8 +1,8 @@
 #include <math.h>
```

Comment:
Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍

Reply:
Alright, I will.